library/items/item_group/src/testmod/java/org/quiltmc/qsl/items/itemgroup/test/ItemGroupTest.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2021 QuiltMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.quiltmc.qsl.items.itemgroup.test;
+
+import org.quiltmc.qsl.itemgroup.api.QuiltItemGroup;
+
+import net.minecraft.item.ItemGroup;
+import net.minecraft.item.ItemStack;
+import net.minecraft.item.Items;
+import net.minecraft.util.Identifier;
+import net.minecraft.util.registry.Registry;
+
+import net.fabricmc.api.ModInitializer;
+
+public class ItemGroupTest implements ModInitializer {
+	public static final String MOD_ID = "qsl_items_item_group_testmod";
```

Comment:
Honestly, in code it's more used as a namespace than as a mod id, mod id is only relevant for stuff that needs the mod container from Quilt Loader.

```suggestion
	public static final String NAMESPACE = "qsl_items_item_group_testmod";
```